utils/cmake/Dreamcast.toolchain.cmake

```diff
@@ -0,0 +1,2 @@
+message(AUTHOR_WARNING "Dreamcast.toolchain.cmake is deprecated, please use kallistios.toolchain.cmake")
+include($ENV{KOS_BASE}/utils/cmake/kallistios.toolchain.cmake)
```

Comment:
`include(kallistios.toolchain)` should work just as well.

Reply:
Ok, I think this is because KallistiOS' cmake modules path is added from the toolchain script itself, so it makes sense that it doesn't work here.